.github/actions/wsl-bash/action.yaml

```diff
@@ -38,7 +38,7 @@ runs:
         if ( "${exit}" -eq "$False" ) { Exit(1) }
 
     - name: Run linux script
-      uses: ./.github/actions/wsl-powershell
+      uses: Ubuntu/WSL/.github/actions/wsl-powershell@main
```

Comment:
Honest question: is this syntax case sensitive? I'd expect it to fail because the org name is capitalized.

Reply:
Apparently not, because the "example action using WSL" passed. I'll still fix it just to make it cleaner,